tests/e2e-pw/specs/shopper/shopper-subscriptions-purchase-multiple-subscriptions.spec.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import test, { Page, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { shouldRunSubscriptionsTests } from '../../utils/constants';
+import { describeif, getMerchant, getShopper } from '../../utils/helpers';
+import RestAPI from '../../utils/rest-api';
+import { config } from '../../config/default';
+import {
+	emptyCart,
+	fillCardDetails,
+	placeOrder,
+	setupProductCheckout,
+} from '../../utils/shopper';
+import {
+	goToShop,
+	goToShopWithCurrency,
+	goToSubscriptions,
+} from '../../utils/shopper-navigation';
+import {
+	activateMulticurrency,
+	deactivateMulticurrency,
+} from '../../utils/merchant';
+
+const products = {
+	'Subscription no signup fee product': 'subscription-no-signup-fee-product',
+	'Subscription signup fee product': 'subscription-signup-fee-product',
+};
+const configBillingAddress = config.addresses.customer.billing;
+let wasMulticurrencyEnabled = false;
+
+describeif( shouldRunSubscriptionsTests )(
+	'Subscriptions > Purchase multiple subscriptions',
+	() => {
+		let merchantPage, shopperPage: Page;
```

Comment:
Nitpick: I think we should change this to:

```
let merchantPage: Page, shopperPage: Page;
```

Otherwise I'm seeing that `merchantPage` implicitly has an `any` type.

Reply:
Yep, that's right, thanks for catching it!

Reply:
Added it in [b038179](https://github.com/Automattic/woocommerce-payments/pull/10170/commits/b0381791fc959bcfd8581631d70b7c06cfd63a2d).